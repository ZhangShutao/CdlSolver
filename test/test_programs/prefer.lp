action(1..3).
1{occur(X):action(X)}1.
safe :- occur(1).
comfort :- occur(2).

% question: how to represent "if and only if?"
-ideal :- not safe.
-ideal :- not comfort.

:- not &c{ideal}, not safe.

#show occur/1.