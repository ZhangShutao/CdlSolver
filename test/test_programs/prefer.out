default model: 1
x:occur(1); y:occur(1);
Satisfiable